docker/container-test.sh

```diff
@@ -11,9 +11,8 @@ function cleanup {
 }
 
 function test_container {
-    local QUERY_TIMEOUT=150
-    local QUERY_PERIOD=15
-    local QUERY_RETRIES=$((QUERY_TIMEOUT/QUERY_PERIOD))
+    local QUERY_PERIOD=5
+    local QUERY_RETRIES=20
```

Comment:
> `Reduce delay when building Docker container`

`Reduce test grace period ...` ?

Reply:
btw `RESULT`, `I` should be `local` too

Reply:
btw is `set +e` needed?
i think `set -e` doesn't have an effect on commands in conditional statements, i expect this applies to `until` as well

Reply:
I titled the commit because the purpose of the change is to reduce the delay by not sleeping 15 seconds after the first failure. I'll increase the retry count to `30` so that the total retry time is unchanged (although I don't think we need that long).

I don't think `set +e` is needed. Let's follow up on that later, though.